elements/pl-matching/pl-matching.py

```diff
@@ -210,15 +210,15 @@ def prepare(element_html, data):
     # Build the options to display to the student.
     chosen_option_names = []
     display_options = []
-    for (i, opt) in enumerate(options):
+    for i, opt in enumerate(options):
```

Comment:
This is actually a pretty cool addition IMO, I always try to shoot for not having the parens here when it's not necessary.